packages/frontend/core/src/modules/peek-view/view/doc-preview/doc-peek-view.css.ts

```diff
@@ -15,7 +15,7 @@ export const editor = style({
 globalStyle(`[data-full-width-layout="true"] ${editor}`, {
   vars: {
     '--affine-editor-width': '100%',
-    '--affine-editor-side-padding': '24px',
```

Comment:
this is intented. 72px is the old value. see PD-1407

Reply:
Previously, the TOC would overlap with the editor content, so the full-width padding was changed to 72px. However, the peekview was not modified, so the overlap still occurs. Therefore, this change is reasonable. @zanwei